homeassistant/components/fan/mqtt.py

```diff
@@ -21,8 +21,9 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.typing import HomeAssistantType, ConfigType
-from homeassistant.components.fan import (SPEED_LOW, SPEED_MEDIUM,
-                                          SPEED_HIGH, FanEntity,
+from homeassistant.components.fan import (SPEED_MINIMUM, SPEED_LOW,
```

Comment:
This fan speed constant doesn't exist in the base component. Changing the base component requires approval in an issue in our architecture repo.